Backoffice/EventSubscriber/AddTransverseBlockSubscriber.php

```diff
@@ -49,7 +49,7 @@ public function addTransverseBlock(BlockNodeEvent $event)
             $transverseNode->addBlock($newBlock);
             $blockIndex = $transverseNode->getBlockIndex($newBlock);
             /** @var AreaInterface $area */
-            $area = $transverseNode->getAreas()->first();
+            $area = $transverseNode->getArea()->getAreas()->first();
```

Comment:
@alavieille : this sounds weird getArea()->getAreas(). I assume the getArea refers to the main root area. If so you could be nice to rename into getRootArea()